### PR TITLE
Feat/277/create background job to delete expired refresh tokens

### PR DESCRIPTION
**Pull request by NMarteniuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streetcode.BLL/Services/BackgroundServices/DeleteExpiredRefreshTokensService.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        return Task.CompletedTask;
+    }
+
+    public void Dispose()
+    {
+        _timer?.Dispose();
+    }
+
+    private async void DoWork(object? state)
+    {
```
Comment (yaninavoda) [inline Streetcode/Streetcode.BLL/Services/BackgroundServices/DeleteExpiredRefreshTokensService.cs:43]: I don't particularly like this method name'DoWork'. Can you rename it to describe what this method really does?

**Review by yaninavoda — APPROVED** [review]
I left 2 comments.

**Review by yaninavoda — COMMENTED** [review]
builed failed

**Review by HlibBondarev — APPROVED** [review]
Ok

**Pull request merged**